### DISCUSSION TITLE
I have problem in responsiveness for drawing the signature

### DISCUSSION THREAD

**Discussion by pmahip** [szimek/signature_pad#793] [General]
I am using desktop, mobile, and tablet views in my application. While the signature works fine on desktop, it overflows on mobile and tablet.

![image](https://github.com/user-attachments/assets/4beee0c8-6c5b-4b56-b5e5-55d1688d03a0)

mobile

![image](https://github.com/user-attachments/assets/1457ec20-4e6e-43be-8e49-2b9529f96ca9)

the signature is overflowing to the bottom and user can't see.

if I check this in demo https://szimek.github.io/signature_pad/ also have this bug.

![image](https://github.com/user-attachments/assets/ef2ff26a-a9e4-4b9d-a69c-476948eef75b)

Please help?

**Comment by UziTech** [comment]
If you are drawing outside the box why shouldn't the signature be outside the box? It seems to me like the desktop version is the one that has the bug. Unless you drew the signature with those sharp corners that doesn't look the same as the shape that was actually drawn.

The desktop was fixed to act like the mobile version in v5 (#765). It looks like your application is not using the latest version.
